src/use-intersection/use-intersection.ts

```diff
@@ -66,7 +66,6 @@ export const useIntersection = (controller: IntersectionComposableController, op
   }
 
   Object.assign(controller, {
-    isVisible: false,
```

Comment:
Should we keep this attribute with something like this?

```js
Object.assign(controller, {
  get isVisible () {
    if (controller.element === targetElement) {
      return controller.element.hasAttribute('isVisible')
    }
  }
})
```

Reply:
Hmm, good idea. But the problem is kinda the same: which element do you ask if it's visible if you have more than one element per controller?

Reply:
That's the reason of the `if`. Only return `true/false` if the target is the controller, `undefined` otherwise.

Also the getter could be override only if the controller is the target.

```js
if (controller.element === targetElement) {
  Object.assign(controller, {
    get isVisible () {
      return controller.element.hasAttribute('isVisible')
    }
  })
}
```

Reply:
Oh, I see - my bad. Yeah that could work.

Reply:
Another idea would to have a "at least one is visible" and/or a "all are visible" helper

Reply:
> Another idea would to have a "at least one is visible" and/or a "all are visible" helper

That will be perfect for my usecase since I'm building a responsive horizontal menu a la GitHub 😆

Reply:
Just added that, how do you feel about it?

Reply:
Yes, that seems too specific for me. I can do it myself in a simple way.

```js
import { Controller } from '@hotwired/stimulus'
import { useIntersection } from 'stimulus-use'

export default class extends Controller {
  static targets = ['item']

  get allVisible () {
    return this.itemTargets.every(item => item.hasAttribute('isVisible'))
  }

  connect () {
    this.itemTargets.forEach(item => useIntersection(this, { element: item }))
  }

  appear () {
    this.update()
  }

  disappear () {
    this.update()
  }

  update () {
    if (this.allVisible) {
      // ...
    }
  }
}
```

Reply:
I'm happy to keep the helper functions in there, I just was just referring to callbacks and that it would be too much too add them as individual ones. 

I'll update the docs and then I think we are good to go!